packages/@aws-cdk-testing/framework-integ/test/aws-events-targets/test/ecs/image-simple/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM --platform=linux/amd64 public.ecr.aws/docker/library/python:3.12
+ADD index.py .
+CMD [ "python3", "./index.py" ]
```

Comment:
this image needed to be changed. it was using a lambda image which resulted in the following:
`entrypoint requires the handler name to be the first argument`.